examples/nlp/dialogue_state_tracking/dialogue_state_tracking_sgd.py

```diff
@@ -175,11 +175,26 @@
     "--debug_mode", action="store_true", help="Enables debug mode with more info on data preprocessing and evaluation",
 )
 
+parser.add_argument(
+    "--embedding_dimension",
+    default=768,
+    type=int,
```

Comment:
we shouldn't hard code this - it's the hidden_size

Reply:
You mean we should hard code it? I mean set it always fixed value of 768?

Reply:
no, it should `not` be hard coded - it can be extracted from model.hidden_size and will work with any model not just bert-base

Reply:
Made it hard coded for now. Lets make them better later.